src/main/java/com/github/forax/umldoc/gen/PlantUmlGenerator.java

```diff
@@ -139,7 +140,8 @@ private void generateMethodCall(Call.MethodCall methodCall, Writer writer)
     // deactivate Bob
   }
 
-  private void generateCalls(Call.Group group, Writer writer) throws IOException {
+  private void generateCalls(Call.Group group, Writer writer)
+          throws IOException {
```

Comment:
what the point of such change ?

Reply:
Once again, my checkstyle gave me a warning because of the length of the line, I will check why